website/versioned_docs/version-0.10.0/gitops-run/overview.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Overview
+sidebar_position: 1
+hide_title: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Gitops Run Overview
+
+## Introduction
+
+GitOps is a powerful mechanism for creating consistent environments and having 
+multiple clusters stay in sync. If you build out your infrastructure correctly 
+you get predictable behaviours for your teams and you can have new environments 
+up and running quickly. However, GitOps can be challenging for the everyday developer 
+to work with and it can create some friction, especially for developers who are 
+less familiar with Kubernetes or Flux. 
+
+The purpose of GitOps run is to remove the complexity for developers so that 
+platform operators can create developer environments easily, and application developers 
+can benefit from GitOps and focus on writing code. 
+
+Watch this video to learn more about how GitOps run can help your team 
+get started with GitOps:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/2TJz7RhDtAc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Additional Benefits
+* No need to run `kubectl`, `helm`, `kustomize`, of `flux` CLI commands.  Just create the manifests and we'll put them on the cluster for you.
+* Reduces the cycle time when configuring your cluster.  With normal gitops there is a lot of commit/push/reconcile workflows that can be frustrated.  This skips that and you can test your changes directly before commiting and pushing code to git.
```

Comment:
* Reduces the cycle time when configuring your cluster. With normal GitOps, there is a lot of commit/push/reconcile workflows that can be frustrated. This skips that and you can test your changes directly before committing and pushing code to your Git repository.